apps/examples/touchscreen/touchscreen_main.c

```diff
@@ -202,6 +202,38 @@ static int touchsceen_resume(void)
 	return OK;
 }
 
+static int touchscreen_enable(void)
+{
+	int fd = open(TOUCH_DEV_PATH, O_RDWR);
+	if (fd < 0) {
+		printf("Fail to open %s, errno:%d\n", TOUCH_DEV_PATH, get_errno());
+		return ERROR;
+	}
+
+	if (ioctl(fd, TSIOC_ENABLE, NULL) != OK) {
+		printf("Fail to TSIOC_ENABLE %s, errno:%d\n", TOUCH_DEV_PATH, get_errno());
+	}
+
+	close(fd);
+	return OK;
```

Comment:
When we get a failure on the open, we return ERROR.
But when we get a fail on the ioctl, we return OK.
How about using a local variable `ret`?